src/index.js

```diff
@@ -49,7 +49,7 @@ function setBuildNumber(usePackageJsonVersion, branch, gitHubToken, gitHubRepo,
 	if (branch == "master")
 	{
 		console.log("Building master");
-		buildNumber = `${ buildNumber }-r${ shortHash }`;
+		buildNumber = `${ buildNumber }+r${ shortHash }`;
```

Comment:
Can you also update the readme section: https://github.com/nice-digital/teamcity-build-number#build-number-format as it's got a master example and _Major.Minor.Patch.Counter-(BranchName OR rCommitHash]_ is no longer valid